content/en/graphing/functions/rollup.md

```diff
@@ -5,17 +5,40 @@ kind: documentation
 
 `.rollup()`
 
-Recommended for expert users only. Datadog rolls up data points automatically, based on the in-app metric type: `gauge` metrics are averaged by default, whereas `count` and `rate` metrics are summed. Appending this function to the end of a query allows you to override the default behavior: to control the rollup method, or the number of raw points rolled up into a single point plotted on the graph.
+*Recommended for expert users only.*
 
-The function takes two parameters, method and time: `.rollup(method,time)`. The method can be sum/min/max/count/avg and time is in seconds. You can use either one individually, or both together like `.rollup(sum,120)`. We impose a limit of 350 points per time range. For example, if you're requesting `.rollup(20)` for a month-long window, we return data at a rollup far greater than 20 seconds in order to prevent returning a gigantic number of points.
+Appending the `.rollup()` function at the end of a query allows you to perform custom [time aggregation][1]. This function enables you to define the time intervals for your graph and how data points are aggregated within a given time interval.
```

Comment:
Hm, can we add to the end of this?

> This function enables you to define the time intervals for your graph and how data points are aggregated within a given time interval.

Instead, can we just mention the limit? I'm afraid of someone referencing the doc and not reading the bottom section, this bit out of context would be puzzling if the rollup was ignored.

Could you get behind:

> This function enables you to define the time intervals for your graph (if larger than the query-enforced rollup interval) and how data points are aggregated within a given time interval.

Something to that effect? Just to call out that there IS a caveat.